Dockerfile

```
@@ -8,11 +8,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+FROM public.ecr.aws/eks-distro/kubernetes/go-runner:v0.13.0-eks-1-23-1 as go-runner
 
-FROM golang:1.15-alpine AS build
-RUN apk --no-cache update && \
-    apk --no-cache add ca-certificates git && \
-    rm -rf /var/cache/apk/*
+FROM public.ecr.aws/bitnami/golang:latest AS build
```

Same here on parameterizing
```suggestion
ARG BUILDER=public.ecr.aws/bitnami/golang:latest
FROM ${BUILDER} AS build
```

---

Also, why are you using bitnami's build of Go?

---

builds fail due to docker rate limiting. However with parameterization, we won;t have to change it here and still be able to use public ecr based images during build